server/listeners/eventHandlers/probationEvents/offenderManagerChangedEventHandler.ts

```diff
@@ -7,8 +7,12 @@ export default class OffenderManagerChangedEventHandler {
 
   handle = async (event: ProbationEventMessage): Promise<void> => {
     const { crn } = event
+    const deliusRecord = await this.communityService.getProbationer({ crn })
```

Comment:
This is the endpoint which is more accurate.. but still need to cross reference with `/allOffenderManagers` because it has the required data fields (username)